Wikipedia/Code/PageHistoryCollectionViewCell.swift

```diff
@@ -16,6 +16,9 @@ class PageHistoryCollectionViewCell: CollectionViewCell {
     var displayTime: String? {
         didSet {
             timeLabel.text = displayTime
+            if let displayTime = displayTime {
```

Comment:
Accessibility label is nullable so I'm not sure this check is necessary. It could lead to a state where the accessibility label doesn't match the actual label when the value is nil'd out

Reply:
oh good catch!